pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -43,6 +45,21 @@ var genericPreflightChecks = [...]PreflightCheck{
 		fix:              fixBundleCached,
 		flags:            SetupOnly,
 	},
+	{
+		configKeySuffix:  "check-ram",
+		checkDescription: "Checking minimum RAM requirements",
+		check:            checkRAM,
+		fixDescription:   "crc requires at least 9GB to run",
+		flags:            NoFix,
+	},
+}
+
+func checkRAM() error {
+	totalMemory := memory.TotalMemory()
+	if totalMemory < 9_000_000_000 {
```

Comment:
@guillaumerose we already have a constant which consist of default memory so shouldn't we use something like

```patch
 func checkRAM() error {
        totalMemory := memory.TotalMemory()
-       if totalMemory < 9_000_000_000 {
+       logging.Debugf("Total memory of system is %d Bytes", totalMemory)
+       if totalMemory < constants.DefaultMemory*1024*1024 {
                return fmt.Errorf("9GB of memory are required to run crc (%s found)", units.HumanSize(float64(totalMemory)))
        }
        return nil
```

Also we should move it so some other package and make it public so even with `crc config set memory <>` we can use this as part of validation https://github.com/code-ready/crc/blob/master/pkg/crc/validation/validation.go#L25

Reply:
While I agree this is already better, it is actually the amount of free memory that prevents Hyper-V to start a VM.

Reply:
@praveenkumar nice! I will change this